src/gsy_e/gsy_e_core/area_serializer.py

```diff
@@ -17,10 +17,12 @@
 """
 import json
 
+from gsy_framework.utils import convert_pendulum_to_str_in_dict, key_in_dict_and_not_none
+
 from gsy_e.models.area import Area # NOQA
 from gsy_e.models.strategy import BaseStrategy
 from gsy_e.models.area.throughput_parameters import ThroughputParameters
-
+# pylint: disable=unused-import
```

Comment:
Two notes:
1. could you please clarify why we need all these unused imports below? I'm not exactly sure I understand the reasons for all these `# NOQA` here.
2. we usually put `# pylint: disable` comments before all imports (when they apply to the entire module) or on the specific line of the warning (if it's just that line).

Reply:
WRT #1, I got curious and thought to try it out. It looks like the NOQA imports are needed for running the gsy-e using a json setup file: 

```
$ gsy-e -l INFO run --setup json_file -t 15s -s 15m  --seed 0 

.......


18:00:24.919 INFO     (  22) root                          : Running without Kafka connection for simulation results.
Traceback (most recent call last):
  File "/Users/spyrostzavikas/dev/gsy-e/src/gsy_e/gsy_e_core/area_serializer.py", line 110, in area_from_dict
    return _leaf_from_dict(description, config)  # Area is a Leaf
  File "/Users/spyrostzavikas/dev/gsy-e/src/gsy_e/gsy_e_core/area_serializer.py", line 97, in _leaf_from_dict
    raise ValueError("Unknown leaf type '%s'" % leaf_type)
ValueError: Unknown leaf type '<class 'NoneType'>'

```

Therefore the deserialization needs these imports. However, from my tests all these imports can be deleted because they essentially duplicate this:
`from gsy_e.models.leaves import *  # NOQA`

which follows some lines below and also duplicates the same strategy imports. Therefore the other strategy imports can be removed (also the Area import as far as I see)